### ISSUE TITLE
ngb-pagination: bootstrap css is not working

### ISSUE THREAD

**Issue by jomark-t** [ng-bootstrap/ng-bootstrap#3674]
I'm working on an angular project with the latest version. I have a page where to display data from a database to a table. I've used the ng-bootstrap table along with its pagination. Everything is working, the data generation, the pagination as well as the item display per page except the styling of ng-pagination is not working. The following classes are `.pagination` `.page-item` and `.page-link`. Those classes can be found in `bootstrap.css`. I've included the latest bootstrap too. I know the `bootstrap.css` is working because the styling for table is working as well as the item per page selection.

I only included the `bootstrap.css` in one component.

I tried overriding it and no avail. Please help, thanks!

```
Angular CLI: 9.0.3
Node: 12.16.1
OS: win32 x64
ng-bootstrap: 6.0.2
Boostrap: 4.4.1
```

**Comment by benouat** [comment]
Thank you @jomark-t for reporting a potential bug.

Nonetheless, according to our documentation [You think you've found a bug](
https://github.com/ng-bootstrap/ng-bootstrap#you-think-youve-found-a-bug) could you please reproduce your scenario in a minimalist Stackblitz in order for us to start working on it ?

**Comment by jomark-t** [comment]
Ok, I'll try to reproduce it thank you.

**Comment by jomark-t** [comment]
@benouat I did reproduced it

https://stackblitz.com/edit/angular-ayshm7?embed=1&file=src/app/app.component.html

**Comment by benouat** [comment]
@jomark-t I think the issue still comes from Bootstrap CSS not being loaded! 😞 

I've seen in your SB that you added in `app.component.css`:

```css
@import '~bootstrap/dist/css/bootstrap.css';
```

**It won't work.** You've misused the `@import` rule here.

In a pure CSS context, it can only be used to reference, and load an existing css file by url. i.e. the file **needs** to be served by your static file server.

Then, in SCSS context, it can be used to import some style from filesystem. The **~** syntax here allows you to load a file from `node_modules` when the sass file is preprocessed at build time.

To prove my point, I simply loaded bootstrap CSS from the index.html instead, and it works right away.

![image](https://user-images.githubusercontent.com/1152740/79312889-dc89cc80-7eff-11ea-8128-b491befd52c4.png)

---
**EDIT**: I just realised that our documentation is wrong here. The code snippet to add the css file in CSS context is wrong. (it's the one you used maybe). We will update that asap.
